cmd/tailscale/cli/debug.go

```diff
@@ -49,18 +49,44 @@ import (
 var debugCmd = &ffcli.Command{
 	Name:       "debug",
 	Exec:       runDebug,
-	ShortUsage: "tailscale debug <debug-flags | subcommand>",
+	ShortUsage: "tailscale debug <subcommand>",
 	ShortHelp:  "Debug commands",
 	LongHelp:   hidden + `"tailscale debug" contains misc debug facilities; it is not a stable interface.`,
-	FlagSet: (func() *flag.FlagSet {
-		fs := newFlagSet("debug")
-		fs.StringVar(&debugArgs.file, "file", "", "get, delete:NAME, or NAME")
-		fs.StringVar(&debugArgs.cpuFile, "cpu-profile", "", "if non-empty, grab a CPU profile for --profile-seconds seconds and write it to this file; - for stdout")
-		fs.StringVar(&debugArgs.memFile, "mem-profile", "", "if non-empty, grab a memory profile and write it to this file; - for stdout")
-		fs.IntVar(&debugArgs.cpuSec, "profile-seconds", 15, "number of seconds to run a CPU profile for, when --cpu-profile is non-empty")
-		return fs
-	})(),
 	Subcommands: []*ffcli.Command{
+		{
+			Name:       "file",
+			ShortUsage: "tailscale debug file --name <get, delete:NAME, or NAME>",
+			Exec:       runFile,
+			ShortHelp:  "Run taildrop file",
+			FlagSet: (func() *flag.FlagSet {
+				fs := newFlagSet("file")
+				fs.StringVar(&fileArgs.name, "name", "", "get, delete:NAME, or NAME")
+				return fs
+			})(),
+		},
+		{
+			Name:       "mem-profile",
+			ShortUsage: "tailscale debug mem-profile --out",
+			Exec:       runPprofMemory,
+			ShortHelp:  "Run pprof memory",
+			FlagSet: (func() *flag.FlagSet {
+				fs := newFlagSet("mem-profile")
+				fs.StringVar(&pprofMemoryArgs.out, "out", "", "grab a memory profile and write it to this file; - for stdout")
+				return fs
+			})(),
+		},
+		{
+			Name:       "cpu-profile",
+			ShortUsage: "tailscale debug cpu-profile --out --profile-seconds",
```

Comment:
example doesn't have an arg for `--out`. And above.

Reply:
Thanks for the review!